test/utils/jasmine-matchers.js

```diff
@@ -122,6 +123,36 @@ exports.execute = function() {
                         };
                     }
                 }
+            },
+
+            toThrow: function(util, customEqualityTesters) {
```

Comment:
@howdyjessie  Yes, it overrides Jasmine's `toThrow` matcher. The difference is between the matching criteria. **Jasmine** matches the whole **exception** object, whereas my method matches it based on either the **errorcode**, **message** or the **exception** object itself.

It still works the same way as it should, with some added benefits.